test/rules/assertions/requireReturnsCheck.js

```diff
@@ -223,6 +223,20 @@ export default {
           }
       `
     },
+    {
+      code: `
+          /**
+           * @interface
+           */
+          class Foo {
+            /**
+             * @returns {string}
+             */
+            bar () {
+            }
+          }
+      `
+    },
```

Comment:
If you can also add a corresponding invalid example (where there is no `@interface`), so we can see the contrast, I think this will be ready to merge.

Reply:
Makes sense, added an invalid test case.